exec/cnex/lib/runtime_win32.c

```diff
@@ -0,0 +1,389 @@
+#include "runtime.h"
+
+#define COBJMACROS
+
+#include <iso646.h>
+#include <windows.h>
+
+#include "assert.h"
+#include "cell.h"
+#include "enums.h"
+#include "exec.h"
+#include "object.h"
+#include "stack.h"
+
+#ifdef _WIN32
```

Comment:
I think you can already assume that this file is being built for win32, no need for an ifdef.

Reply:
Fixed.